docs/_source/components-overview/pop_syn/single_star.rst

```diff
@@ -21,85 +23,167 @@ STARPROPERTIES
+      
+      * velocity
+      * theta
+      * phi
+      * ?
+
+  * - ``SN_type``
     - The supernova type of the star.
-  * - `f_fb`
+  * - ``f_fb``
     - The fraction of fallback mass.
-  * - `spin_orbit_tilt_first_SN`
+  * - ``spin_orbit_tilt_first_SN``
     - The spin-orbit tilt after the first SN, if the star has undergone a SN.
-  * - `spin_orbit_tilt_second_SN`
+  * - ``spin_orbit_tilt_second_SN``
     - The spin-orbit tilt after the second SN, if a second SN has occurred.
-  * - `m_disk_radiated`
+  * - ``m_disk_radiated``
     - The mass of the disk radiated in the collapse of the star.
+  * = ``m_disk_accreted``
+    - The mass of the disk accreted in the collapse of the star.
 
 
-TODO: add missing properties
-
 State options
 ~~~~~~~~~~~~~
 
-Star states are defined on their relative position on the HR diagram as follows:
-
-.. csv-table:: State
-   :header: "State", "Description"
-   :widths: 10, 30
-
-   `PreMS`, "The star is on the pre-main-sequence phase."
-   `MS`, "The star is in the Main Sequence phase."
-   `PostMS`, "The star is in the post Main Sequence phase."
-   `HeMS`, "The star is at the beginning of the He burning phase."
-   `PostHeMS`, "The star is evolving through the beginning of the He burning phase."
-   `WD`, "The star is a White Dwarf."
-   `NS`, "The star is a Neutron Star."
-   `BH`, "The star is a Black Hole."
-
-TODO: update the states to POSYDON v2.0.0
+Star states are defined by their burning and surface properties.
+These states are combined to describe the stellar state.
+We also have additional extra states for object that are not stars.
+
+.. list-table:: Surface state
+  :header-rows: 1
+  :widths: 10 30
+
+  * - State
+    - Description
+  * - ``H-rich``
+    - The star has a hydrogen-rich surface.
+  * - ``stripped_He``
+    - The star has a stripped helium surface.
+  * - ``accreted_He``
+    - The star has an accreted helium rich material on its surface.
+  
+.. list-table:: Burning state
+  :header-rows: 1
+  :widths: 10 30
+
+  * - State
+    - Description
+  * - ``non_burning``
+    - The star is not burning.
+  * - ``Core_H_burning``
+    - The star is burning hydrogen in its core.
+  * - ``Shell_H_burning``
+    - The star is burning hydrogen in its shell.
+  * - ``Core_He_burning``
+    - The star is burning helium in its core.
+  * - ``Central_He_depleted``
+    - The star has depleted helium in its core.
+  * - ``Shell_He_burning``
+    - The star is burning helium in its shell.
+  * - ``Core_C_burning``
+    - The star is burning carbon in its core.
+  * - ``Central_C_depletion``
+    - The star has depleted carbon in its core.
+
+.. list-table:: Additional States
+  :header-rows: 1
+  :widths: 10 30
+
+  * - State
+    - Description
+  * - ``WD``
+    - The star is a White Dwarf.
+  * - ``NS``
+    - The star is a Neutron Star.
+  * - ``BH``
+    - The star is a Black Hole.
+  * - ``massless_remnant``
+    - The star exploded or merged and only a single star is left. 
```

Comment:
Here you used the word star twice but referred to different objects. Hence, I'd suggest to use star and companion, e.g.
"The star exploded or merged and only its companion is left as a single star."